src/cluster/slurm.py

```diff
@@ -5,50 +5,53 @@
 
```

Comment:
same thing about spacing as comment above 🤞

Reply:
Yes, thank you for being a stickler.

`black` was applied to this file for conformance to the SSE coding standards.

Expect a dedicated PR in the near future exclusively dedicated to formatting.